CasperSleep/nightshade
# Swap normalize.css -> normalize.scss

Another small post-deployment optimization pulled out into smaller commit. 
1. Removes `normalize.css` as dependency and from `base_layout.html`
2. Add `normalize.scss`—fork of the package in casper org repo—and import in `application.scss`

Sister commit https://github.com/CasperSleep/storefront-static/pull/42
### Looking for feedback
1. Should  `normalize.scss` be a dependency of `nightshade-core`? 
2. Should we continue to shrinkwrap devDeps?
### Testing
1. Clean-up and load new dependencies

```
npm prune
npm install
```
1. Run `gulp sass`. Compilation should complete without any errors
2. Run `gulp`. Site should not look drastically different from http://nightshade.rocks
